test/test_string.erl

```diff
@@ -0,0 +1,9 @@
+-module(test_string).
+
+-export([f/0, g/0]).
+
+%% test case for string literal (ASCII)
+f() -> "abc%/_hoge".
+
+%% test case for string literal (japanese)
+g() -> "いろはにほへとちるぬるを".
```

Comment:
Sorry, could you add a test case for empty string (`""`)?

Reply:
yes.

Reply:
Thank you!